staging/src/k8s.io/apimachinery/pkg/runtime/serializer/allocator_test.go

```diff
@@ -0,0 +1,71 @@
+package serializer
+
+import (
+	"crypto/rand"
+	"io/ioutil"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	testapigroupv1 "k8s.io/apimachinery/pkg/apis/testapigroup/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/runtime/serializer/protobuf"
+)
+
+func BenchmarkProtobufEncoder(b *testing.B) {
```

Comment:
Actually, these aren't really benchmarks of allocator, but rather benchmark of serializer.
Maybe move them to something like "encoder_with_allocator_test.go" or sth?

Reply:
sure, I can move to a new file.